bundles/org.openhab.binding.gpio/src/main/resources/ESH-INF/thing/gpio-digital-output-thing.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="gpio"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="gpio-digital-output">
```

Comment:
Apply formatting on the file  (Ctrl-Shift-F in eclipse) to correct indentation and using tabs everywhere.